db-persistence/my/SetupDatabase.ps1

```diff
@@ -58,22 +49,20 @@ if ($restartingInstance) {
                 }
 
                 $_.Alter()
-                $_.SetOffline()
+                try {
+                    $_.SetOffline()
+                } catch {
+                    # ignore
```

Comment:
I know why you do it like this, but would it maybe make sense to at least check if it is offline afterwards?

Reply:
Then the `Move-Item` would fail because the file is in use, isn't that enough to stop the execution or should we introduce another error?

Reply:
I just wanted a nicer "oops, this didn't work, have to stop" message instead of waiting that it crashes with a somewhat obscure error message

Reply:
Ok, added the warning. It's printed for every sandbox image but like I mentioned before, it works anyway.